### ISSUE TITLE
Error with PartitionCells

### ISSUE THREAD

**Issue by Noodlotje** [cole-trapnell-lab/monocle-release#229]
I'm running Monocle3 on R (on MacOS) with the vignet data. R session info pasted at bottom.
When doing reduceDimension I get a warning message:
> cds <- reduceDimension(cds, reduction_method = 'UMAP')
/Library/Python/2.7/site-packages/umap/spectral.py:229: UserWarning: Embedding a total of 2 separate connected components using meta-embedding (experimental)
  n_components
And when trying to proceed to partitioning I get an error that I cannot figure out:
> cds <- partitionCells(cds)
Error in .Call("_monocle_pnorm_over_mat", PACKAGE = "monocle", R_num_links_ij,  : 
  "_monocle_pnorm_over_mat" not available for .Call() for package "monocle"

R session info:
R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.1

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] bindrcpp_0.2.2        monocle_2.99.2        L1Graph_0.1.1         lpSolveAPI_5.5.2.0-17
 [5] DDRTree_0.1.5         irlba_2.3.2           igraph_1.2.2          Biobase_2.42.0       
 [9] DelayedArray_0.8.0    BiocParallel_1.16.0   IRanges_2.16.0        S4Vectors_0.20.1     
[13] BiocGenerics_0.28.0   matrixStats_0.54.0    Seurat_2.3.4          Matrix_1.2-15        
[17] cowplot_0.9.3         ggplot2_3.1.0        

loaded via a namespace (and not attached):
  [1] reticulate_1.10          R.utils_2.7.0            tidyselect_0.2.5        
  [4] htmlwidgets_1.3          grid_3.5.1               combinat_0.0-8          
  [7] trimcluster_0.1-2.1      docopt_0.6.1             Rtsne_0.15              
 [10] devtools_2.0.1           munsell_0.5.0            codetools_0.2-15        
 [13] ica_1.0-2                future_1.10.0            miniUI_0.1.1.1          
 [16] withr_2.1.2              colorspace_1.3-2         fastICA_1.2-1           
 [19] knitr_1.20               rstudioapi_0.8           ROCR_1.0-7              
 [22] robustbase_0.93-3        dtw_1.20-1               pbmcapply_1.3.0         
 [25] listenv_0.7.0            gbRd_0.4-11              labeling_0.3            
 [28] Rdpack_0.10-1            lars_1.2                 slam_0.1-43             
 [31] bit64_0.9-7              pheatmap_1.0.10          rhdf5_2.26.0            
 [34] rprojroot_1.3-2          LearnBayes_2.15.1        coda_0.19-2             
 [37] diptest_0.75-7           R6_2.3.0                 doParallel_1.0.14       
 [40] VGAM_1.0-6               manipulateWidget_0.10.0  hdf5r_1.0.1             
 [43] flexmix_2.3-14           bitops_1.0-6             assertthat_0.2.0        
 [46] promises_1.0.1           SDMTools_1.1-221         scales_1.0.0            
 [49] nnet_7.3-12              gtable_0.2.0             npsurv_0.4-0            
 [52] globals_0.12.4           processx_3.2.0           rlang_0.3.0.1           
 [55] splines_3.5.1            lazyeval_0.2.1           acepack_1.4.1           
 [58] checkmate_1.8.5          rgl_0.99.16              reshape2_1.4.3          
 [61] crosstalk_1.0.0          backports_1.1.2          httpuv_1.4.5            
 [64] Hmisc_4.1-1              tools_3.5.1              usethis_1.4.0           
 [67] spData_0.2.9.4           gplots_3.0.1             RColorBrewer_1.1-2      
 [70] proxy_0.4-22             sessioninfo_1.1.1        ggridges_0.5.1          
 [73] Rcpp_1.0.0               plyr_1.8.4               base64enc_0.1-3         
 [76] purrr_0.2.5              densityClust_0.3         ps_1.2.1                
 [79] prettyunits_1.0.2        deldir_0.1-15            rpart_4.1-13            
 [82] pbapply_1.3-4            viridis_0.5.1            zoo_1.8-4               
 [85] ggrepel_0.8.0            cluster_2.0.7-1          fs_1.2.6                
 [88] magrittr_1.5             data.table_1.11.8        gmodels_2.18.1          
 [91] lmtest_0.9-36            RANN_2.6                 mvtnorm_1.0-8           
 [94] fitdistrplus_1.0-11      pkgload_1.0.2            lsei_1.2-0              
 [97] mime_0.6                 xtable_1.8-3             mclust_5.4.1            
[100] sparsesvd_0.1-4          gridExtra_2.3            HSMMSingleCell_1.2.0    
[103] compiler_3.5.1           tibble_1.4.2             KernSmooth_2.23-15      
[106] crayon_1.3.4             R.oo_1.22.0              htmltools_0.3.6         
[109] spdep_0.7-9              segmented_0.5-3.0        later_0.7.5             
[112] Formula_1.2-3            snow_0.4-3               tidyr_0.8.2             
[115] expm_0.999-3             MASS_7.3-51.1            fpc_2.1-11.1            
[118] boot_1.3-20              cli_1.0.1                R.methodsS3_1.7.1       
[121] gdata_2.18.0             metap_1.0                bindr_0.1.1             
[124] pkgconfig_2.0.2          sp_1.3-1                 foreign_0.8-71          
[127] plotly_4.8.0             foreach_1.4.4            webshot_0.5.1           
[130] bibtex_0.4.2             stringr_1.3.1            callr_3.0.0             
[133] digest_0.6.18            tsne_0.1-3               htmlTable_1.12          
[136] DelayedMatrixStats_1.4.0 curl_3.2                 kernlab_0.9-27          
[139] shiny_1.2.0              gtools_3.8.1             modeltools_0.2-22       
[142] nlme_3.1-137             jsonlite_1.5             Rhdf5lib_1.4.0          
[145] desc_1.2.0               viridisLite_0.3.0        limma_3.38.2            
[148] pillar_1.3.0             lattice_0.20-38          httr_1.3.1              
[151] DEoptimR_1.0-8           pkgbuild_1.0.2           survival_2.43-1         
[154] glue_1.3.0               remotes_2.0.2            qlcMatrix_0.9.7         
[157] FNN_1.1.2.1              png_0.1-7                prabclus_2.2-6          
[160] iterators_1.0.10         glmnet_2.0-16            bit_1.1-14              
[163] HDF5Array_1.10.0         class_7.3-14             stringi_1.2.4           
[166] mixtools_1.1.0           doSNOW_1.0.16            latticeExtra_0.6-28     
[169] caTools_1.17.1.1         memoise_1.1.0            dplyr_0.7.8             
[172] ape_5.2

**Comment by chatterjee89** [comment]
Any luck on this?

**Comment by Noodlotje** [comment]
Nope, none. I have temporarily given up on monocle, assuming it is caused
by the cycle of updates in R, hoping that if I need it again, updates are
in phase again...

Op wo 10 apr. 2019 om 18:35 schreef Deeptiman Chatterjee <
notifications@github.com>:
